crates/red_knot_python_semantic/src/types.rs

```diff
@@ -296,6 +297,42 @@ impl<'db> Type<'db> {
         }
     }
 
+    /// Return true if this type is assignable to type `other`.
```

Comment:
I would remove the comment because it doesn't provide more information than the function signature or expand it to explain what assignability means.

Reply:
Maybe a better comment would link to the typing spec section on assignability